client/client.go

```diff
@@ -105,7 +106,9 @@ func (r *NotaryRepository) Initialize(rootKeyID string) error {
 		return err
 	}
 
-	rootCert, err := cryptoservice.GenerateCertificate(privKey, r.gun)
+	// Hard-coded policy: the generated certificate expires in 10 years.
+	startTime := time.Now()
+	rootCert, err := cryptoservice.GenerateCertificate(privKey, r.gun, startTime, startTime.AddDate(10, 0, 0))
```

Comment:
The ambition of this PR is to make functions available for testing; making the hard-coded policy more visible is mostly a side effect. Making the policy configurable should IMHO go into a separate PR, if it needs to be configurable at all (I don’t currently see a really strong reason for it to be configurable, OTOH I haven’t been thinking about it much.)

(Also, `Time.Add(hours)` does not account for leap years the way `Time.AddDate` does, see 856f9021c8f08e9e6c7c7d43b9cec64d531a3605 . That’s certainly not a showstopper but it does lead to non-intuitive and surprising results.)